Resources/Locale/en-US/reagents/meta/consumable/drink/alcohol.ftl

```diff
@@ -55,6 +55,9 @@ reagent-desc-whiskey = A type of distilled alcoholic beverage made from fermente
 reagent-name-wine = wine
 reagent-desc-wine = An premium alchoholic beverage made from distilled grape juice.
 
+reagent-name-champagne = champagne
+reagent-desc-champagne = An premium sparkling wine
```

Comment:
Shouldn't this be "A premium sparkling wine" instead?

Reply:
> Shouldn't this be "A premium sparkling wine" instead?

Oops

Reply:
done